helpers/slots.js

```diff
@@ -15,13 +15,23 @@
 'use strict';
 
 var constants = require('./constants.js');
+
 /**
- * @memberof module:helpers
- * @module helpers/slots
+ * Description of the module.
+ *
+ * @module
+ * @requires helpers/constants
+ * @property {number} interval - Slot time interval in seconds.
+ * @property {number} delegates - Active delegates from constants.
+ * @see Parent: {@link helpers}
+ * @todo Add description of the module
  */
+
 /**
  * Gets constant time from Lisk epoch.
- * @returns {number} epochTime from constants.
+ *
+ * @private
```

Comment:
Are these definitely private?

Reply:
I would say yes, as they are not exported by the module.